hydpy/core/hydpytools.py

```diff
@@ -236,7 +238,7 @@ class `HydPy`, and the |hland_derived.UH| instance `uh` of application model
     >>> classname(hp.nodes)
     'Nodes'
 
-    >>> classname(model.parameters.derived.uh)
+    >>> classname(model.rconcmodel.parameters.control.uh)
```

Comment:
Could then be switched to `Z`, as well. 

Please note the text before these doctests needs updating (e.g. hland_derived.UH`)